admin_interface/templates/admin/base_site.html

```diff
@@ -9,12 +9,9 @@
 {% block extrastyle %}
 {% get_admin_interface_theme as theme %}
 {% get_admin_interface_version as version %}
+{% get_admin_interface_nocache as version_md5_cache %}
 {% get_current_language as current_lang %}
 <style type="text/css">
-/*
-django-admin-interface {{ version }}
-https://github.com/fabiocaccamo/django-admin-interface
-*/
```

Comment:
The version could be removed without deleting the whole comment

Reply:
I will re-add it in the future if I will add the version somewhere.